pkg/controllers/validationController_test.go

```diff
@@ -98,15 +98,15 @@ func TestValidateRequestBody(t *testing.T) {
 	assert.Contains(t, strings.TrimSpace(responseRecorder.Body.String()), "We're good!")
 }
 
-func TestValidateRequestBodyWithNotAllowedK8sResource(t *testing.T) {
+func TestValidateRequestBodyWithAllowedK8sResource_kubectlApply(t *testing.T) {
 	request := httptest.NewRequest(http.MethodPost, "/validate", strings.NewReader(applyRequestNotAllowedJson))
 	request.Header.Set("Content-Type", "application/json")
 	responseRecorder := httptest.NewRecorder()
 
 	validationController := NewValidationController()
 	validationController.Validate(responseRecorder, request)
 
-	assert.Contains(t, strings.TrimSpace(responseRecorder.Body.String()), "\"allowed\":false")
+	assert.Contains(t, strings.TrimSpace(responseRecorder.Body.String()), "\"allowed\":true")
```

Comment:
The reason I flipped this test case is that I think it should actually be skipped, notice the
```json
"manager":"kubectl-client-side-apply"
```
![Screen Shot 2022-09-21 at 3 15 15](https://user-images.githubusercontent.com/39004075/191387192-b5703e5b-56ed-457a-92d0-ff7c5f6dc05b.png)